.travis.yml

```diff
@@ -91,11 +91,20 @@ before_install:
     - pwd
     - mkdir /tmp/cozy-desktop
     - ln -s /tmp/cozy-desktop tmp
+    - npm install -g yarn
```

Comment:
Automatic yarn install [confirmed](https://travis-ci.org/cozy-labs/cozy-desktop/jobs/186096259#L1548-L1598).

Waiting for the `gui/` stuff.